src/commands/commerce/extension/map.ts

```diff
@@ -68,7 +67,8 @@ export class MapExtension extends SfdxCommand {
         if (storeName === undefined && storeId === undefined) {
             throw new SfdxError(msgs.getMessage('extension.map.undefinedName'));
         }
-        const storeid = this.validateStoreId(storeName, storeId, userName);
+        const validateId = new UtilStoreValidate();
```

Comment:
nit: utility classes are usually not instantiated and contain only static methods which will work in this case. Would we consider following the format like jsonUtils.ts where they export the function instead?